Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -482,15 +482,15 @@ def rasterize_command(urls: Union[list[str], str], rasterize_timeout: int) -> li
         }
     )
     demisto.debug(f'Rasterize Data: {res_rasterize}')
-    return [res['Contents'] for res in res_rasterize]
+    return [res['Contents'] for res in res_rasterize][:len(urls)]
 
 
 def rasterize_urls(urls: list[str], rasterize_timeout: int) -> list[dict]:
     res_rasterize = rasterize_command(urls, rasterize_timeout)
     if len(res_rasterize) < len(urls):  # check for errors in the response
         demisto.info(f'Rasterize response is too short, running command for each URL\n{res_rasterize=}\n{urls=}')
         rasterize_runs = map(rasterize_command, urls, [rasterize_timeout] * len(urls))
-        res_rasterize = sum(rasterize_runs, [])
+        res_rasterize = [run[0] for run in rasterize_runs]
```

Comment:
this is no longer a number, on purpose? Explain this change

Reply:
It never was a number it was always a list. In this case I'm just picking out the first of each so as not to take more than one response per URL.